cirq-core/cirq/experiments/xeb_sampling_test.py

```diff
@@ -35,7 +34,7 @@ def test_sample_2q_xeb_circuits():
             q0,
             q1,
             depth=20,
-            two_qubit_op_factory=lambda a, b, _: SQRT_ISWAP(a, b),
+            two_qubit_op_factory=lambda a, b, _: ops.SQRT_ISWAP(a, b),
```

Comment:
```suggestion
            two_qubit_op_factory=lambda a, b, _: cirq.SQRT_ISWAP(a, b),
```

Reply:
@tanujkhattar, sorry to bother you again.  But I have some questions.
As per the https://github.com/quantumlib/Cirq/issues/4223#issuecomment-863703185 it was mentioned:
> Within cirq-core, use ops.SQRT_ISWAP and use cirq.SQRT_ISWAP elsewhere (cirq.ops is used to avoid circular dependencies within cirq-core)
So, won't this create a circular dependency?  I am not sure if the latest changes have something to do with this.

Reply:
Use `cirq.SQRT_ISWAP` in tests, because tests are not imported anywhere else and hence we directly `import cirq` in tests. For other files in cirq-core, import ops and use `ops. SQRT_ISWAP` (for files within the ops/* directy you'd directly have to import the file instead of importing module ops).